### DISCUSSION TITLE
Steady State Solution Iterated with Trasient Solution.

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#20222] [Q&A Modules: General] [answered]
Is it possible to solve steady state and transient problems in a singe .i file? The physics for each are different (steady state flow, transient heat transfer) but parameters can be shared from each solution type to the other?

**Comment by GiudGiud** [comment, marked as answer]
Hello

Yes but it's kind of hacky.
If you 'forget' the time derivatives for some variables, they will be solved for their steady state value.
Numerically, there is no guarantee it will work. But it's been done before for flow and heat conduction.

Guillaume

**Reply by TLWise** [reply]
I expect that I could put the solvers in sequential order, i.e. steady state first, then transient to gain the sequential solutions? Are there any example problems for this? Thank you.

**Reply by GiudGiud** [reply]
no they will be solved simultaneously.
If you want to order solves you ll have to use the Multiapps system, which wont use the same .i file.
That is the cannon way of doing this.
